### PR TITLE
update to 1.16

### PR DESCRIPTION
**Pull request by LegendaryGeek**

### CODE REVIEW

File: src/main/java/com/builtbroken/woodenshears/loot/LootModifierWoodenShears.java [hunk 1]
```diff
@@ -48,4 +48,6 @@ private LootContext createContext(LootContext context)
                 .withParameter(LootParameters.TOOL, new ItemStack(Items.SHEARS))
                 .build(LootParameterSets.BLOCK);
     }
+
+
```
Comment (bl4ckscor3) [inline src/main/java/com/builtbroken/woodenshears/loot/LootModifierWoodenShears.java]: Unnecessary new lines

File: src/main/java/com/builtbroken/woodenshears/loot/LootModifierWoodenShearsSerializer.java [hunk 2]
```diff
@@ -15,4 +15,6 @@ public LootModifierWoodenShears read(ResourceLocation location, JsonObject objec
     {
         return new LootModifierWoodenShears(conditions);
     }
+
+
```
Comment (bl4ckscor3) [inline src/main/java/com/builtbroken/woodenshears/loot/LootModifierWoodenShearsSerializer.java]: Unnecessary new lines

**Review by bl4ckscor3 — CHANGES_REQUESTED** [review]
1. Please remove `CREDITS.txt`, `LICENSE.txt`, `README.txt`, and `changelog.txt`. Those are Forge-specific files and have nothing to do with this project. We also already have all of those files (changelog is in the readme) specific to our project.
2. The game doesn't start with this mod installed. Either you didn't test your changes (which you should have done) or you did not commit the changes to `mods.toml`.

**Comment by LegendaryGeek** [comment]
should be fixed now

**Comment by bl4ckscor3** [comment]
Still got an extra empty line at each place where you removed one (I marked two lines in both my comments because there are two lines to remove each!) Also since you've change the loader version dependency, might as well change the forge dependecy to `[32,)` as well. Code looks good otherwise, ~~will test soon~~mod works as well. Will merge after the two lines are removed and the version is changed as explained above :) Thank you very much for your contribution!

**Pull request merged**